workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -654,7 +791,7 @@ describe('DataSetService', () => {
       jest.spyOn(WbcDataSetsAuthorizationPlugin.prototype, 'getAccessPermissions').mockResolvedValue({
         data: {
           dataSetId: mockDataSetId,
-          permissions: [{ identity: mockUserId, identityType: 'USER', accessLevel: 'read-write' }]
+          permissions: [{ identity: mockUserId, identityType: 'USER', accessLevel: 'read-only' }]
```

Comment:
This shouldn't stay `read-write`?

Reply:
I changed it to better encompass all possible combinations of permissions and identity types